src/styles/utils/mixin.scss

```diff
@@ -0,0 +1,23 @@
+@mixin number-day {
+  @for $cell from 1 through 31 {
+    & :nth-child(#{$cell})::before {
+      content: '#{$cell}';
+    }
+  }
+}
+
+@mixin weak-day-lentgh {
+  @each $week-day, $index in $start-day {
+    &--start-day-#{$week-day} :nth-child(1) {
+      margin-left: $cellSize * $index + $index;
```

Comment:
Ensure that the variable '$start-day' is defined in 'variables.scss' or within the same file before using it in a mixin. If it's not defined, it will cause a compilation error.